src/Files.App/Views/MainPage.xaml

```diff
@@ -215,78 +215,81 @@
+					<uc:InnerNavigationToolbar
+						x:Name="InnerNavigationToolbar"
+						Grid.ColumnSpan="3"
+						x:Load="False"
+						Loaded="NavToolbar_Loaded"
+						ShowPreviewPaneButton="{x:Bind ViewModel.ShouldPreviewPaneBeDisplayed, Mode=OneWay}"
+						ShowViewControlButton="{x:Bind ViewModel.ShouldViewControlBeDisplayed, Mode=OneWay}"
+						TabIndex="2" />
 
-				<!--  Page Content  -->
-				<ContentPresenter
-					Grid.Row="1"
-					Grid.Column="0"
-					HorizontalAlignment="Stretch"
-					HorizontalContentAlignment="Stretch"
-					Content="{x:Bind ((viewmodels:MainPageViewModel)DataContext).SelectedTabItem.ContentFrame, Mode=OneWay}" />
+					<!--  Page Content  -->
+					<ContentPresenter
+						Grid.Row="1"
+						Grid.Column="0"
+						HorizontalAlignment="Stretch"
+						HorizontalContentAlignment="Stretch"
+						Content="{x:Bind ((viewmodels:MainPageViewModel)DataContext).SelectedTabItem.ContentFrame, Mode=OneWay}" />
 
-				<!--  Preview Pane Splitter  -->
-				<toolkit:GridSplitter
-					x:Name="PaneSplitter"
-					Grid.Row="1"
-					Grid.Column="1"
-					x:Load="{x:Bind ViewModel.ShouldPreviewPaneBeActive, Mode=OneWay}"
-					ManipulationCompleted="PaneSplitter_ManipulationCompleted"
-					ManipulationStarted="PaneSplitter_ManipulationStarted"
-					ResizeBehavior="BasedOnAlignment"
-					Style="{StaticResource DefaultGridSplitterStyle}" />
+					<!--  Preview Pane Splitter  -->
+					<toolkit:GridSplitter
+						x:Name="PaneSplitter"
+						Grid.Row="1"
+						Grid.Column="1"
+						x:Load="{x:Bind ViewModel.ShouldPreviewPaneBeActive, Mode=OneWay}"
+						ManipulationCompleted="PaneSplitter_ManipulationCompleted"
+						ManipulationStarted="PaneSplitter_ManipulationStarted"
+						ResizeBehavior="BasedOnAlignment"
+						Style="{StaticResource DefaultGridSplitterStyle}" />
 
-				<!--  Preview Pane  -->
-				<uc:InfoPane
-					x:Name="PreviewPane"
-					Grid.Row="1"
-					Grid.Column="2"
-					HorizontalContentAlignment="Stretch"
-					x:Load="{x:Bind ViewModel.ShouldPreviewPaneBeActive, Mode=OneWay}"
-					Loaded="PreviewPane_Loaded"
-					Unloaded="PreviewPane_Unloaded" />
-
-				<!--  Status Bar  -->
-				<uc:StatusBar
-					x:Name="StatusBar"
-					Grid.Row="4"
-					Grid.ColumnSpan="3"
-					x:Load="False"
-					ShowInfoText="{x:Bind SidebarAdaptiveViewModel.PaneHolder.ActivePaneOrColumn.InstanceViewModel.IsPageTypeNotHome, Mode=OneWay}" />
-
-			</Grid>
+					<!--  Preview Pane  -->
+					<uc:InfoPane
+						x:Name="PreviewPane"
+						Grid.Row="1"
+						Grid.Column="2"
+						HorizontalContentAlignment="Stretch"
+						x:Load="{x:Bind ViewModel.ShouldPreviewPaneBeActive, Mode=OneWay}"
+						Loaded="PreviewPane_Loaded"
+						Unloaded="PreviewPane_Unloaded" />
+				</Grid>
+			</sidebar:SidebarView.InnerContent>
 
+			<!--  Footer Content  -->
+			<sidebar:SidebarView.FooterContent>
+				<Border>
+					<!--  Status Bar  -->
+					<uc:StatusBar
+						x:Name="StatusBar"
+						Height="32"
```

Comment:
Could this be a problem with larger text sizes and text getting clipped? (Not like this wouldn't have been an issue without this PR)

Reply:
I haven’t seen any reports of this nature, but like you said, it wouldn’t be a new issue if that’s the case…